CMakeLists.txt

```diff
@@ -37,6 +37,7 @@ add_subdirectory(sdk/platform/noplatform)
 add_subdirectory(sdk/platform/posix)
```

Comment:
Were the file mode changes here, and some of the other files in this PR, intentional?

https://stackoverflow.com/questions/16748160/what-does-it-mean-when-git-diff-shows-mode-changes/16748188

Reply:
No they were not. Thanks for the comment. I will revert the modes.

Reply:
Done. Thanks.